modules/virt-example-snippets-nmstate-IP-management.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-snippets-nmstate-IP-management_{context}"]
+= Example snippets: IP management
```

Comment:
Remove "snippets" and change the title to "Examples: IP management"?

Reply:
+1, I think that we can just say "Example" in the titles as long as we make it clear in the body of the module that this is a snippet.

Reply:
Dropped from both files